extra-i18n/fcitx5-chinese-addons/autobuild/defines

```diff
@@ -0,0 +1,4 @@
+PKGNAME=fcitx5-chinese-addons
+PKGDES=""Addons related to Chinese, including IME previous bundled inside fcitx4""
+PKGDEP="curl libime qt-5 fcitx5-qt boost opencc fcitx5"
```

Comment:
I don't see why it needs cURL. Explain your reason.

Reply:
my fault.....I see PKGBUILD, but i have no tested...

Reply:
Okay, looks like the cloud Pinyin plugin needs `cURL`